front/pages/api/w/[wId]/log.ts

```diff
@@ -0,0 +1,54 @@
+import { NextApiRequest, NextApiResponse } from "next";
+
+import { Authenticator, getSession } from "@app/lib/auth";
+import logger from "@app/logger/logger";
+import { apiError, withLogging } from "@app/logger/withlogging";
+
+async function handler(
```

Comment:
I'm not a big fan of this. But I can live with it, but it needs more work:
- Need to have contextual information added from what we know server side from the auth object (workspaceId, etc...)
- Need to add support for optionally incrementing a counter because this is how we graph stuff and we'd want to have it here for this use-case
- Add a library client-side to call into that with best-effort, we're not going to do fetch(...) everytime we want to log something from the client side.